recorder/sandbox.rb

```diff
@@ -46,6 +46,16 @@
   File.read(File.join('..', 'extension/fonts/glyphicons-halflings-regular.woff'))
 end
 
+get "/extension/fonts/fontawesome-webfont.ttf?v=4.2.0" do
+  content_type 'application/font-woff'
+  File.read(File.join('..', 'extension/fonts/fontawesome-webfont.ttf'))
+end
+
+get "/extension/fonts/fontawesome-webfont.woff?v=4.2.0" do
+  content_type 'application/font-woff'
+  File.read(File.join('..', 'extension/fonts/fontawesome-webfont.woff'))
+end
```

Comment:
I'm not sure it'd help too much. Like download size is not a huge issue. With that said I included both cause I don't have a strong handle on font sources, so happy for some more expertise.